src/main/java/step4/view/InputView.java

```diff
@@ -0,0 +1,45 @@
+package step4.view;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class InputView {
+    public static final String FIRST_QUESTION = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분";
+    public static final String SECOND_QUESTION = "시도할 회수는 몇 회 인가요?";
+    public static final String WRONG_ARGUMENT = "잘못된 인자입니다.";
+    public static final String QUESTION_NUMBER_ONE = "1";
+    public static final String QUESTION_NUMBER_TWO = "2";
+    private Scanner scanner = new Scanner(System.in);
```

Comment:
```suggestion
    private static final Scanner SCANNER= new Scanner(System.in);
```
매번 객체생성시 새로 만들기보다는 캐싱하는것이 좋을것 같아요 !